plugins/compress/compress.cc

```diff
@@ -70,7 +70,7 @@ const int BROTLI_LGW               = 16;
 
 static const char *global_hidden_header_name = nullptr;
 
-static TSMutex compress_config_mutex = TSMutexCreate();
+static TSMutex compress_config_mutex = nullptr;
```

Comment:
Was this change (deferring the initialization to `TSPluginInit`) supposed to be included?

Reply:
Addressed in PR description.